packages/hash/README.md

```diff
@@ -16,6 +14,10 @@ HASH is an open-source, data-centric, all-in-one workspace. HASH combines a rich
 
 We will be developing HASH into a production-grade application which can be self-hosted. The current design and architecture, while not fully realized, paves the way for further features, scale, and performance. You can read about the long-term vision for HASH [here](https://hash.ai/platform/hash).
 
+> **Warning:**
+> The repository is currently in a state of flux while some large improvements are being implemented.
+> As such, portions of this README may prove outdated in the interim, this could include guides on how to load blocks, references to various services, broken tests, features, etc.
```

Comment:
@Alfred-Mountfield I would prefer we be more specific here, if possible, but this is fine.

Reply:
I'd err on leaving it as is for now, for similar reasoning as Ciaran:
> I agree that the level of flux means it will be difficult to keep this in sync and risks ending up with a README that incorrectly flags things as broken which aren't, or vice versa.
>
> We should have a separate record of 'things in the readme that are broken' which we can keep track of and tick off as we fix them (e.g. in Asana).

If we end up rethinking this then we can update as needed, I just think things are going to be changing a **lot** and it'll be a bit hard to keep the README up to date. That said, we'll need to restructure the asana soon, so perhaps that would be a good opportunity to encapsulate and write it down here for open-consumption.